test/controllers/RegistrationControllerSpec.scala

```diff
@@ -1584,10 +1586,31 @@ class RegistrationControllerSpec extends PAYERegSpec with AuthFixture with Regis
       when(mockSubmissionService.submitTopUpToDES(ArgumentMatchers.any(), ArgumentMatchers.any())(ArgumentMatchers.any[HeaderCarrier]()))
```

Comment:
change this test to read "return a 500 response when the registration we try to incorporate is in invalid status and maxIIcount < config value"